### DISCUSSION TITLE
about  initialize_weights

### DISCUSSION THREAD

**Discussion by wssfan** [ultralytics/yolov5#5674] [Q&A]
I am a beginner in DL,   Recently I‘ve do a little change in the yolov5's structure, than i should initialize the training models, then saw the function about initialize_weights,  why the func is nn.Conv2d,  then then pass? and   I saw the initial values  fall  into a very big values,it's about 0.45. Why?

**Comment by glenn-jocher** [comment]
@wssfan weight initialization is here, you're free to modify as you see fit:
https://github.com/ultralytics/yolov5/blob/47fac9ff73aceedd267db1e734a98de122fc9430/utils/torch_utils.py#L155-L165
